README.md

```diff
@@ -0,0 +1,51 @@
+# Auth2 client for Python
+
+This repo contains a minimal client for the [KBase Auth2 server](https://github.com/kbase/auth2),
+covering only the most common operations - e.g. validating tokens and user names
+and getting user roles.
+
+Most other uses are easily done with any http/REST client like `requests` or `httpx`.
+
+## Installation
+
+TODO INSTALL setup a KBase pypi org and publish there
```

Comment:
please announce when you do this as there are a few other packages that it'd be great to put up there